05-JWT-Basics/preferred/middleware/authenticate.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken')
+
+const authenticationMiddleware = async (req, res, next) => {
+  const authHeader = req.header('Authorization')
+
+  if (!authHeader || !authHeader.startsWith('Bearer ')) {
+      res.status(401).json({message: 'Unauthorized - no token provided'})
+    }
+    const token = authHeader.split(' ')[1]
+    //console.log(token);
+
+    try {
+      const name_of_user = jwt.verify(token, process.env.jwt_secret)
```

Comment:
Change the jwt_secret to uppercase. Example: process.env.JWT_SECRET

Reply:
Done